environments/editors.md

```diff
@@ -49,6 +49,14 @@ launch the editor from the dashboard.
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
+### System requirements
+
+For the best possible experience it is recommended to run the editor in an
+environment with:
+
+- at least 8 Gb of RAM
+- a CPU with at least 4 cores
```

Comment:
I think that different wording would be better. Maybe something like
```
- At least 4 CPU cores
```

Reply:
alternatively, we could say:

For the best possible experience it is recommended to run the editor in an
environment with at minimum the following resources:

-8GB RAM
- 4 CPU cores

Reply:
Done.